content/blog/2018-02-07-react-v-16-3.md

```diff
@@ -3,7 +3,7 @@ title: "React v16.3.0: New lifecycles and context"
 author: [bvaughn]
 ---
 
-This release includes a new class component lifecycle (`getDerivedStateFromProps`), a new `StrictMode` component, and an official context API!
+This release includes a new class component lifecycle (`getDerivedStateFromProps`), a new `StrictMode` component, an official context API, and a better way for managing refs!
```

Comment:
“Better” might imply you have to convert from functional refs.

Maybe “and a new ergonomic ref API”.

Reply:
I dig it.